src/DependenciesPackager/CrossgenContext.cs

```diff
@@ -181,10 +182,13 @@ private bool CrossgenAssembly(string publishFolder, PackageEntry package, Librar
 
         private DirectoryInfo CreateSubDirectory(string cacheBasePath, PackageEntry package, LibraryAsset asset)
         {
+            // special treatment of the cases in the path to accomodating NuGet package search
+            // https://github.com/aspnet/BuildTools/issues/94
+            // https://github.com/NuGet/Home/issues/2522
             var subDirectoryPath = Path.Combine(
                 cacheBasePath,
-                package.Library.Identity.Name,
-                package.Library.Identity.Version.ToNormalizedString(),
+                package.Library.Identity.Name.ToLower(),
```

Comment:
These should be `ToLowerInvariant`, since that's the method NuGet uses to lowercase these thingies.